### PR TITLE
feat: move GET events by interest route under experimental

### PR DESCRIPTION
**Pull request by dav1do**
Linear ticket WS1-1522. 

Created a new GET /experimental/events/{sep}/{model} route to query events by an interest filter. The old route still exists (marked deprecated) until the tests are updated and then it can be deleted.

Targets #285 which targets #284

### CODE REVIEW

File: api/ceramic.yaml [hunk 1]
```diff
@@ -156,9 +156,69 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
+  "/experimental/events/{sep}/{model}":
```
Comment (AaronGoldman) [inline api/ceramic.yaml:159]: /experimental/events/{sep_key}/{sep_value}

File: api/ceramic.yaml [hunk 2]
```diff
@@ -156,9 +156,69 @@ paths:
...
+    get:
+      summary: Get events matching the interest stored on the node
+      parameters:
+        - name: sep
+          in: path
+          description: name of the separator/sort_key e.g. 'model'
```
Comment (AaronGoldman) [inline api/ceramic.yaml:165]: Name of the field in the Events header that holds the separator value.

File: api/ceramic.yaml [hunk 3]
```diff
@@ -156,9 +156,69 @@ paths:
...
+          schema:
+            type: string
+          required: true
+        - name: model
+          in: path
+          description: value associated with the sort key e.g. multibase encoded model ID
```
Comment (AaronGoldman) [inline api/ceramic.yaml:171]: The value of the field in the Events header indicated by the separator key.

File: api/ceramic.yaml [hunk 4]
```diff
@@ -156,9 +156,69 @@ paths:
...
+    get:
+      summary: Get events matching the interest stored on the node
+      parameters:
+        - name: sep
+          in: path
+          description: name of the separator/sort_key e.g. 'model'
+          schema:
+            type: string
+          required: true
+        - name: model
```
[the remark this answers is not in the document]
Comment (dav1do) [reply api/ceramic.yaml]: yeah, this one was based on #285. I addressed it in that one but hadn't merged it here yet. Should be updated now.. and hopefully I got some of the other sep/sepValue renames as well.

File: api/src/metrics/api.rs [hunk 5]
```diff
@@ -73,6 +73,26 @@ where
         .await
     }
 
+    /// Get events for a stream
+    async fn experimental_events_sep_model_get(
+        &self,
+        sort_key: String,
```
[the remark this answers is not in the document]
Comment (dav1do) [reply api/src/metrics/api.rs]: updated

File: api/ceramic.yaml (hunk 2 above)
Comment (dav1do) [reply api/ceramic.yaml]: updated

File: api/ceramic.yaml (hunk 3 above)
Comment (dav1do) [reply api/ceramic.yaml]: updated

**Review by nathanielc — APPROVED** [review]

**Pull request merged**